e2e_tests/cli/test_import.py

```diff
@@ -78,15 +80,55 @@ def assert_same_annotation_properties(
             assert expected_property in actual_properties  # type : ignore
 
 
+def get_base_slot_of_item(
+    config_values: ConfigValues, dataset_id: int, item_idx: int
+) -> str:
+    """
+    Returns the base slot for the nth item in a specific `E2EDataset`. The base slot is
+    always the first listed slot
```

Comment:
When we talk about base slot for layers, we should retrieve it from the layout, not actual item slots. Unless I'm missing something.